crates/forge/src/init.rs

```diff
@@ -0,0 +1,115 @@
+use anyhow::{bail, Context, Ok, Result};
+
+use include_dir::{include_dir, Dir};
+use scarb_metadata::MetadataCommand;
+
+use std::fs;
+use std::path::{Path, PathBuf};
+use std::process::{Command, Stdio};
+use toml_edit::{ArrayOfTables, Document, Item, Table};
+
+static TEMPLATE: Dir = include_dir!("starknet_forge_template");
+
+fn overwrite_files_from_template(
+    dir_to_overwrite: &str,
+    base_path: &PathBuf,
+    project_name: &str,
+) -> Result<()> {
+    let copy_from_dir = TEMPLATE.get_dir(dir_to_overwrite);
```

Comment:
```suggestion
    let copy_from_dir = TEMPLATE.get_dir(dir_to_overwrite).ok_or_else(|| anyhow!(...))?;
```

Nit but this way we don't have to deal with match below